src/blocks/sound.rs

```diff
@@ -408,8 +408,11 @@ impl PulseAudioClient {
 
                 loop {
                     // make sure mainloop dispatched everything
-                    for _ in 0..10 {
+                    loop {
```

Comment:
Did you run into an issue with this, or was this just something you noticed in the code? (No idea where the magic number came from in the original)

Reply:
For me the loop always runs just once. Block works as expected. I suppose this is just the "correct" way to do this.

Reply:
No, I didn't run into an issue, but I changed it while debugging. It didn't make sense to run the loop an arbitrary number of times instead of just making sure that the context state is ready.